django_select2/admin.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""
+Django-Select2 Tools for Django Admin.
+
+    Sample usage:
```

Comment:
Since I accepted this correction, I had not commented on it. Maybe this way it looks like I am still in doubt about it. I prefer pushing all changes together but, if you say so, I will first push this smaller fixes first.